index.html

```diff
@@ -81,38 +84,42 @@ <h2>
 }
 </script>
 
-<h2>
-<a id="installation" class="anchor" href="#installation" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Installation (Micro service)</h2>
+            <h2>
+                <a id="installation" class="anchor" href="#installation" aria-hidden="true"><span aria-hidden="true"
+                    class="octicon octicon-link"></span></a>Installation (Micro service)
+            </h2>
 
-<p>Start the self executing JAR</p>
+            <p>Start the self executing JAR</p>
 <pre>java -jar generate-cli.jar</pre>
 
 <p>or deploy it in Tomcat.</p>
 
-<h2>
-<a id="usage-cli-command-line-interface" class="anchor" href="#usage-cli-command-line-interface" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Usage CLI (Command Line Interface).</h2>
+            <h2>
+                <a id="usage-cli-command-line-interface" class="anchor" href="#usage-cli-command-line-interface"
+                    aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Usage CLI (Command Line Interface)
+            </h2>
 
-<p>Examples used here are for the next generation of Eiffel events called semantics which is open source. Please follow <a href="https://github.com/Ericsson/eiffel/tree/master/eiffel-syntax-and-usage">eiffel syntax</a> and <a href="https://github.com/Ericsson/eiffel/tree/master/eiffel-vocabulary">eiffel vocabulary</a></p>
+            <p>Examples used here are for the next generation of Eiffel events called semantics which is open source. Please follow <a href="https://github.com/Ericsson/eiffel/tree/master/eiffel-syntax-and-usage">eiffel syntax</a> and <a href="https://github.com/Ericsson/eiffel/tree/master/eiffel-vocabulary">eiffel vocabulary</a></p>
 
 <pre>java -jar generate-cli.jar -h
 You passed help flag.
 usage: java -jar
  -d,--debug                  enable debug traces
  -f,--content_file           message content file
  -h,--help                   show help.
- -json,--json_content        json content
+ -json,--json_content        JSON content
  -mp,--messaging_protocol    name of messaging protocol to be used,
                              e.g. eiffel3, semantics
```

Comment:
Is the protocol name not eiffelsemantics, not semantics?

Reply:
yes Lars it is eiffelsemantics only, I will do modify